api/tests/unit/features/test_unit_features_models.py

```diff
@@ -692,6 +697,24 @@ def test_feature_state_value_get_skip_create_audit_log_if_environment_feature_ve
     assert feature_state.feature_state_value.get_skip_create_audit_log() is True
 
 
+def test_feature_state_value_get_skip_create_audit_log_for_deleted_feature_state(
```

Comment:
Based on the codecov report, it looks like this isn't actually testing the codepath that you expect.